TGPControls/TGPCamelLabels7.h

```diff
@@ -48,4 +48,6 @@
 @property (nonatomic, strong) NSArray * names; // Will dictate the number of ticks
 @property (nonatomic, assign) NSTimeInterval animationDuration;
 
+@property (nonatomic, assign) BOOL animate; // Keep labels at original position when selected
```

Comment:
param name and comment is inverted. 
Should be either
1) animate // Make the labels animate when selected 
2) dontAnimate // Keep labels at original position when selected